pkg/agent/openflow/pipeline.go

```diff
@@ -23,26 +24,32 @@ import (
 
 	"github.com/vmware-tanzu/antrea/pkg/agent/config"
 	"github.com/vmware-tanzu/antrea/pkg/agent/openflow/cookie"
+	"github.com/vmware-tanzu/antrea/pkg/agent/proxy/upstream"
 	"github.com/vmware-tanzu/antrea/pkg/agent/types"
 	binding "github.com/vmware-tanzu/antrea/pkg/ovs/openflow"
 )
 
 const (
 	// Flow table id index
-	classifierTable       binding.TableIDType = 0
-	spoofGuardTable       binding.TableIDType = 10
-	arpResponderTable     binding.TableIDType = 20
-	conntrackTable        binding.TableIDType = 30
-	conntrackStateTable   binding.TableIDType = 31
-	dnatTable             binding.TableIDType = 40
-	egressRuleTable       binding.TableIDType = 50
-	egressDefaultTable    binding.TableIDType = 60
-	l3ForwardingTable     binding.TableIDType = 70
-	l2ForwardingCalcTable binding.TableIDType = 80
-	ingressRuleTable      binding.TableIDType = 90
-	ingressDefaultTable   binding.TableIDType = 100
-	conntrackCommitTable  binding.TableIDType = 105
-	l2ForwardingOutTable  binding.TableIDType = 110
+	classifierTable              binding.TableIDType = 0
+	spoofGuardTable              binding.TableIDType = 10
+	arpResponderTable            binding.TableIDType = 20
+	serviceHairpinTable          binding.TableIDType = 29
+	conntrackTable               binding.TableIDType = 30
+	conntrackStateTable          binding.TableIDType = 31
+	sessionAffinityTable         binding.TableIDType = 40
+	dnatTable                    binding.TableIDType = 40
```

Comment:
Is it by intention to share same table for sessionAffinityTable and dnatTable?

Reply:
Yes, I keep this in order to not touch the existing pipeline.